config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -589,6 +591,7 @@ periodics:
           - --
           - --check-leaked-resources
           - --env=KUBE_FEATURE_GATES=DisableCloudProviders=false,DisableKubeletCloudCredentialProviders=false
```

Comment:
i would have expected these to be true

Reply:
i don't want to change the feature gates, we just have to set the cloud provider where the current set works